app/views/user_personalities/_form.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="valign-wrapper row login-form">
+  <div class="col card hoverable s10 pull-s1 m6 pull-m3 l4 pull-l4">
+    <div class="card-content">
+      <span class="card-title center"><%= @personality.id.nil? ? 'Create personality' : 'Update personality' %></span>
+      <div class="divider"></div>
+      <%= render 'shared/error_messages', obj: @personality %>
+      <%= form_with(model: @personality, local: true) do |f| %>
+        <div class="form-inputs row">
+          <div class="input-field col s12">
+            <%= f.text_field :name, required: true, autofocus: true, placeholder: "Wife" %>
```

Comment:
why "wife"?

Reply:
cuz its first thing that comes to my head when some male talks about withdrawals